example/package.json

```diff
@@ -10,18 +10,24 @@
   "author": "Dylan Barrell (dylan@barrell.com)",
   "license": "MIT",
   "dependencies": {
-    "axe-core": "^2.6.1",
+    "axe-core": "^3.0.0-alpha.9",
     "browserify": "^16.0.0",
-    "react": "^15.3.0",
-    "react-dom": "^15.3.0"
+    "create-react-class": "^15.6.3",
+    "document-ready-promise": "^3.0.1",
```

Comment:
Are you sure about this dependency?

Reply:
I removed it. But I needed a new release of axe-core to finish this work